apps/artemis-web/pages/bgpupdates.tsx

```diff
@@ -75,4 +77,8 @@ const BGPUpdates: React.FunctionComponent<{}> = () => {
   );
 };
 
-export default BGPUpdates;
+export async function getServerSideProps(ctx) {
+  return getProps(ctx);
+}
+
+export default withAuth(BGPUpdates);
```

Comment:
withAuth should have some additional parameters to pass which roles can access this page:

```
withAuth(BGPUpdates, ACL=['user','admin'])
```
or something like that

Reply:
Oh nice suggestion!